release.ps1

```diff
@@ -5,15 +5,29 @@ else {
     $dir = $PSScriptRoot + "\bin\"
 }
 
-$copy = $dir + "\copy\BepInEx" 
+$out = $dir + "out\"
+$copy = $dir + "copy\"
+$BIEdir = $dir + "BepInEx\"
+$Coredir = $dir + "Core\"
+$UMMdir = $dir + "UMM\"
 
-$ver = "v" + (Get-ChildItem -Path ($dir + "\BepInEx\") -Filter "*.dll" -Recurse -Force)[0].VersionInfo.FileVersion.ToString()
+$ver = (Get-Item ($Coredir + "\RuntimeUnityEditor.Core.dll")).VersionInfo.FileVersion.ToString()
 
-New-Item -ItemType Directory -Force -Path ($dir + "\out")  
+New-Item -ItemType Directory -Force -Path ($out)  
 
-Remove-Item -Force -Path ($dir + "\copy") -Recurse -ErrorAction SilentlyContinue
-New-Item -ItemType Directory -Force -Path ($copy + "\plugins\RuntimeUnityEditor")
-Copy-Item -Path ($dir + "\BepInEx\plugins\") -Destination ($copy) -Recurse -Force 
-Compress-Archive -Path $copy -Force -CompressionLevel "Optimal" -DestinationPath ($dir + "out\" + "RuntimeUnityEditor_BepInEx5_" + $ver + ".zip")
+Remove-Item -Force -Path ($copy) -Recurse -ErrorAction SilentlyContinue
+New-Item -ItemType Directory -Force -Path ($copy)
+Copy-Item -Path ($BIEdir) -Destination ($copy) -Recurse -Force
```

Comment:
I think output dll of the BepInEx5 csproj should be moved directly into $BIEdir and the release script adjusted as such. There's little reason to keep it as it is now since I don't plan on putting any files outside of the plugins directory.
```suggestion
Copy-Item -Path ($BIEdir) -Destination ($copy + "BepInEx\plugins\RuntimeUnityEditor") -Recurse -Force
```

Reply:
I see what you're going for, but currently it's line 20 and 26 that are consistent, 21 is infact the inconsistency. Also line 20 
`Copy-Item -Path ($BIEdir) -Destination ($copy) -Recurse -Force` takes care of building the recursive folrder structure of `BepInEx/plugins/RuntimeUnityEditory`

Because the core is shared, it's the inconsistent element leading to the difference in line 21 and 27.
The consistency IMHO was that BepinEx requires a folder structure of BepInEx/plugins and UMM requires the files are in the root of the zip (the zip is the container), so the build outputs match their required output structure, Core then flexes on 21/27 to match the respective structure.

I'm happy to make the change though if you still want to proceed.

Reply:
I see what you mean, in that case I'll keep it as it is, at least for now.